helpers/interactions/sendInteractionResponse.ts

```diff
@@ -36,7 +36,7 @@ export async function sendInteractionResponse(
 
   // A reply has never been send
   if (bot.cache.unrepliedInteractions.delete(id)) {
-    return await bot.rest.sendRequest<undefined>(bot.rest, {
+    return <undefined> await bot.rest.sendRequest(bot.rest, {
```

Comment:
why?

Reply:
The generic annotation in `bot.rest.sendRequest()` is redundant, since the value is irrelevant, however the function requires that an `undefined` be returned if there is no value, so cast Promise return type to `undefined`.

Reply:
I'm not a fan of this style. I prefer the older way its much better on the eyes and readability. the generic undefined passed to sendRequest tells typescript that it is returning undefined which is exactly what we need imo. No?

Reply:
Indeed, and yes, perhaps it is indeed better for readability. I'll update the code.